libMBIN/Source/NMS/GameComponents/GcStatsEnum.cs

```diff
@@ -3,14 +3,14 @@
 
 namespace libMBIN.NMS.GameComponents
 {
-	[NMS(Size = 0x4, GUID = 0xF28A9BF702A385FF, NameHash = 0xACDE6E8DA19EC2E2)]
+	[NMS(Size = 0x4, GUID = 0x0DF6ED3E1829721E, NameHash = 0xACDE6E8DA19EC2E2)]
     public class GcStatsEnum : NMSTemplate
     {
-        // 0xC entries
-		public enum StatEnum {
-            None, DEPOTS_BROKEN, FPODS_BROKEN, PLANTS_PLANTED, SALVAGE_LOOTED, TREASURE_FOUND, QUADS_KILLED, WALKERS_KILLED, FLORA_KILLED, PLANTS_GATHERED,
-            BONES_FOUND, C_SENT_KILLS
+        // 0xD entries
+		public enum GcStatEnumEnum {
+            None, DEPOTS_BROKEN, FPODS_BROKEN, PLANTS_PLANTED, SALVAGE_LOOTED, TREASURE_FOUND, QUADS_KILLED, WALKERS_KILLED, FLORA_KILLED,
+            PLANTS_GATHERED, BONES_FOUND, C_SENT_KILLS, STORM_CRYSTALS
         }
-		public StatEnum Stat;
+		public GcStatEnumEnum GcStatEnum;
```

Comment:
why do HG do this?? haha

Reply:
and that is not the only one they changed like that...